theme/utils/class-excerpt.php

```diff
@@ -50,8 +47,13 @@ public static function get_excerpt( $source = null, $limit = null ) {
       $output = substr( $output, 0, strripos( $output, ' ' ) );
     }
 
-    $output = '<p>' . $output . '...</p>';
+    // Reduce string to limit.
+    $output = wp_trim_words( $output, $limit, '...' );
+
+    $output = '<p>' . $output . '</p>';
```

Comment:
Should we force a paragraph here? What if someone wants to use `h3` for their excerpts? Maybe we should remove this and let the actual output in the templates worry about this?

Thoughts @infinum/eightshift-team ?

Reply:
agreed 👍

Reply:
WordPress wraps its output in paragraph tags. Both `the_excerpt()` and `get_the_excerpt()` functions do this.

Reply:
Personally, I'm for removing paragraph tags. More than once I've had issues with styling excerpt and needing to reference the actual paragraph to style it properly.

Reply:
Yeah, but we are changing the workings of the excerpt function so we can control it. I'm for removal 🙂